models/user.go

```diff
@@ -1284,45 +1269,50 @@ type SearchUserOptions struct {
 	OrderBy  string
 	Page     int
 	PageSize int // Can be smaller than or equal to setting.UI.ExplorePagingNum
+	IsActive util.OptionalBool
+}
+
+func (opts *SearchUserOptions) toConds() builder.Cond {
```

Comment:
nit: since this returns a `Cond`, please call it `toCond()`.

Reply:
@ethantkoenig For `SearchUserOptions` contains not only 1 condition it should be `toConds`. And `builder.Cond` maybe one condition or composite conditions, so `toConds` will be more suitable.